docs/src/main/asciidoc/extension-cookbook.adoc

```diff
@@ -0,0 +1,179 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Extension Cookbook
+
+:numbered:
+:sectnums:
+:sectnumlevels: 4
+:toc:
+
+include::./attributes.adoc[]
+
+== Context & Objective
+
+The purpose of this cookbook is to explain for extension developer, a step by step process to build an extension from scratch.
+Other guide are API oriented and this one will not detailed API. For this purpose use other guides such as :
```

Comment:
This sentence is confusing

Reply:
is it better with
It do not explain the quarkus API in detail. For this purpose use other guides such as :